test-config/wdio.browserstack.conf.js

```diff
@@ -9,27 +9,25 @@ const commonCapabilities = {
   'browserstack.appium_version': '1.14.0'
 }
 
+// https://www.browserstack.com/automate/capabilities
 const allCapabilities = [
-  { ...commonCapabilities, browserName: 'Chrome', browser_version: '89.0', os: 'Windows', os_version: '10' },
-  { ...commonCapabilities, browserName: 'Chrome', browser_version: '74.0', os: 'Windows', os_version: '10' },
+  { ...commonCapabilities, browserName: 'Chrome', browser_version: 'latest', os: 'Windows', os_version: '10' },
   { ...commonCapabilities, browserName: 'Chrome', browser_version: '40.0', os: 'Windows', os_version: '7' },
-  { ...commonCapabilities, browserName: 'firefox', browser_version: '87.0', os: 'Windows', os_version: '10' },
-  { ...commonCapabilities, browserName: 'firefox', browser_version: '67.0', os: 'Windows', os_version: '10' },
+  { ...commonCapabilities, browserName: 'firefox', browser_version: 'latest', os: 'Windows', os_version: '10' },
   { ...commonCapabilities, browserName: 'firefox', browser_version: '52.0', os: 'Windows', os_version: '7' },
-  { ...commonCapabilities, browserName: 'Safari', browser_version: '14.0', os: 'OS X', os_version: 'Big Sur' },
+  { ...commonCapabilities, browserName: 'Safari', browser_version: '14.1', os: 'OS X', os_version: 'Big Sur' },
   { ...commonCapabilities, browserName: 'Safari', browser_version: '13.1', os: 'OS X', os_version: 'Catalina' },
-  { ...commonCapabilities, browserName: 'Safari', browser_version: '12.0', os: 'OS X', os_version: 'Mojave' },
+  { ...commonCapabilities, browserName: 'Safari', browser_version: '12.1', os: 'OS X', os_version: 'Mojave' },
   { ...commonCapabilities, browserName: 'Safari', browser_version: '11.1', os: 'OS X', os_version: 'High Sierra' },
   { ...commonCapabilities, browserName: 'Safari', browser_version: '10.1', os: 'OS X', os_version: 'Sierra' },
-  { ...commonCapabilities, browserName: 'Safari', browser_version: '7.1', os: 'OS X', os_version: 'Mavericks' },
-  { ...commonCapabilities, browserName: 'IE', browser_version: '11.0', os: 'Windows', os_version: '7' },
-  { ...commonCapabilities, browserName: 'IE', browser_version: '10.0', os: 'Windows', os_version: '7' },
-  { ...commonCapabilities, browserName: 'IE', browser_version: '9.0', os: 'Windows', os_version: '7' },
-  { ...commonCapabilities, browserName: 'Edge', browser_version: '18.0', os: 'Windows', os_version: '10' },
-  { ...commonCapabilities, browserName: 'Edge', browser_version: '15.0', os: 'Windows', os_version: '10', 'browserstack.selenium_version': '3.141.59' },
+  { ...commonCapabilities, browserName: 'IE', browser_version: '11.0', os: 'Windows', os_version: '10' },
+  { ...commonCapabilities, browserName: 'Edge', browser_version: 'latest', os: 'Windows', os_version: '10', 'browserstack.use_w3c': true },
```

Comment:
the edge doesn't work with the "legacy" protocol, so the "w3c" is enabled. https://github.com/webdriverio/webdriverio/issues/3802
I didn't enable it for all the browsers, cause it fails in the older iphones.